packages/lexical-playground/src/ui/DropDown.tsx

```diff
@@ -132,8 +136,12 @@ function DropDownItems({
 
     if (highlightedItem && highlightedItem.current) {
       highlightedItem.current.focus();
+    } else if (!items) {
+      if (dropDownRef && dropDownRef.current) {
```

Comment:
I understand you're just copying code from elsewhere but these ref objects are always defined (at least when using typescript), no reason to check
```suggestion
      if (dropDownRef.current) {
```